wrapanapi/systems/virtualcenter.py

```diff
@@ -243,10 +284,132 @@ def _get_resource_pool(self, resource_pool_name=None):
             return self.system.get_obj(vim.ResourcePool, self.system.default_resource_pool)
         return self.system.get_obj_list(vim.ResourcePool)[0]
 
-    def _clone(self, destination, resourcepool=None, datastore=None, power_on=True,
-               sparse=False, template=False, provision_timeout=1800, progress_callback=None,
+    def _get_cluster_compute_resource(self, cluster_compute_resource_name=None):
+        """ Returns a Compute Cluster Resource managed object. If a name is specified,
+        a vim.ClusterComputeResource object is returned for the specific resource. If no name if
```

Comment:
fix small typo `If no name if` -> `If no name is`

Reply:
:+1: